movingpandas/trajectory_smoother.py

```diff
@@ -0,0 +1,194 @@
+from copy import copy
+from abc import ABC, abstractmethod
+
+import numpy as np
+from shapely.geometry import Point
+
+try:
+    from stonesoup.types.detection import Detection
+    from stonesoup.models.transition.linear import CombinedLinearGaussianTransitionModel, ConstantVelocity
+    from stonesoup.models.measurement.linear import LinearGaussian
+    from stonesoup.predictor.kalman import KalmanPredictor
+    from stonesoup.updater.kalman import KalmanUpdater
+    from stonesoup.types.array import CovarianceMatrix, StateVector
+    from stonesoup.types.prediction import GaussianStatePrediction
+    from stonesoup.initiator.simple import SimpleMeasurementInitiator
+    from stonesoup.types.hypothesis import SingleHypothesis
+    from stonesoup.smoother.kalman import KalmanSmoother
+    from stonesoup.reader.base import DetectionReader
+    from stonesoup.base import Property
+    from stonesoup.buffered_generator import BufferedGenerator
+except ImportError as error:
+    raise ImportError(
+        "Missing optional dependencies. To use the smoother classes please install movingpandas along with the "
+        "'smoothing' optional dependencies (e.g. pip install 'movingpandas[smoothing]')."
```

Comment:
Since MovingPandas cannot be pip installed on Windows, we should tell users how to do this with conda

Reply:
I am not entirely sure how this could be done in conda. I think for optional dependcies in conda you have to define a metapackage, e.g. movingpandas-extras, that contains the extra dependencies. 

How do you intend to enable users to install optional dependencies? I could, for the time being, include instructions for installing stonesoup directly (as I have done in the docstring for the KalmanSmooterCV class). 

Having said that, how do you suggest I proceed?

Reply:
You're right, let's keep it simple and just point to https://stonesoup.readthedocs.io/en/latest/#installation. Users can then just follow the conda install instructions there.